core-bundle/src/Resources/contao/library/Contao/Dbafs.php

```diff
@@ -830,11 +830,6 @@ public static function shouldBeSynchronized($strPath)
 	 */
 	protected static function isFileSyncExclude($strPath)
 	{
-		if (Config::get('uploadPath') == 'templates')
-		{
-			return true;
-		}
-
```

Comment:
Is it safe to remove this?

Reply:
Yes, because the method now uses *contao.upload_path* which is never set to `"templates"` and therefore now works correctly even if `Config::get('uploadPath') == 'templates'` is true.

Reply:
So now the method is executed until line 858 instead of returning early?

Reply:
Yes, and it now works correctly (meaning calling the method on an DBAFS path acually returns `true`).